toolkit/scripts/build_tag_imagecustomizer.mk

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Azure Linux Image Customizer version.
+# This is using semantic versioning.
+#
+# image_customizer_version should have the format:
+#
+#   <major>.<minor>.<patch>
+#
+# and should hold the value of the next (or current) official release, not the previous official
+# release.
+image_customizer_version ?= 0.3.0
+IMAGE_CUSTOMIZER_VERSION_PREVIEW ?= -dev.$(DATETIME_AS_VERSION)+$(GIT_COMMIT_ID)
```

Comment:
Thanks for the update. I'm guessing you want `IMAGE_CUSTOMIZER_VERSION_PREVIEW ` to be configurable by the user. Can you only extend the documentation to mention this argument?

Reply:
Fixed.